### ISSUE TITLE
Images referenced in SASS are not building properly 

### ISSUE THREAD

**Issue by edwardvercoe** [WeAreAthlon/frontend-webpack-boilerplate#38]
Demo page for example - heading.png is not loading correctly which is referenced by demo.scss
images referenced are located within src/images/design folder as per documentation

**Comment by dholewa** [comment]
Hi, i have same issue.

**Comment by pnikolov** [comment]
Hey, I've been able to reproduce the issue, so I am currently in the process of investigating it.

You can use older release of the boilerplate.

**Comment by oli-laban** [comment]
I fixed this by changing (in `webpack.config.js`):

```js
{
  test: /\.(png|gif|jpe?g|svg)$/i,
  use: [
    {
      loader: 'url-loader',
      options: {
        name: 'images/design/[name].[hash:6].[ext]',
        publicPath: '../',
        limit: environment.limits.images,
      },
    },
  ],
},
```

to

```js
{
  test: /\.(png|gif|jpe?g|svg)$/i,
  type: 'asset',
  parser: {
    dataUrlCondition: {
      maxSize: environment.limits.images,
    },
  },
  generator: {
    filename: 'images/design/[name].[hash:6][ext]',
  },
},
```

**Comment by pnikolov** [comment]
Thanks @oli-laban!

The newly released [Release v5.10.0](https://github.com/WeAreAthlon/frontend-webpack-boilerplate/releases/tag/v5.10.0) fixes the loading of images and fonts. The [demo page](https://weareathlon.github.io/frontend-webpack-boilerplate/) is updated to verify the proper loading of assets.
